src/runtime_src/core/include/windows/types.h

```diff
@@ -18,5 +18,6 @@
 #define core_include_windows_types_h_
 
 typedef size_t ssize_t;
+typedef int pid_t;
```

Comment:
Going forward, I thought we were to use `using` instead of `typedef`

Reply:
Nope. this is C code.